pkgs/development/tools/jdt-language-server/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, jdk15
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jdt-language-server";
+  version = "1.5.0";
+  timestamp = "202110191539";
+
+  src = fetchurl {
+    url = "https://download.eclipse.org/jdtls/milestones/${version}/jdt-language-server-${version}-${timestamp}.tar.gz";
+    sha256 = "02acc7zrn3mw39w747km8g3q6y60sg7174l0c0rsbn624zhjqjpx";
```

Comment:
```suggestion
    sha256 = "Lwwo3+wxeiaOxEkEQgZXGBtDp7oqMvC/eI6jiNrLhVI=";
```

Reply:
Strange, I have a different sha256: `0ll5rgd8i8wfg2zz0ciapakl66qqaw344129qj72cyiixkgjh31g`

I've been using this since around Dec 19th